packages/api/helper/mailer.js

```diff
@@ -0,0 +1,45 @@
+const nodemailer = require('nodemailer');
+const smtpTransport = require('nodemailer-smtp-transport');
+
+module.exports = {
+  sendVerification: (nomination) => {
+    var transport = {
+      port: 587,
+      secure: false,
+      requireTLS: true,
+      service: 'gmail',
+      host: 'smtp.gmail.com',
+      auth: {
+        user: process.env.MAILER_USER,
+        pass: process.env.MAILER_PASS,
+      },
+    };
+
+    var transporter = nodemailer.createTransport(smtpTransport(transport));
+
+    transporter.verify((error, success) => {
+      if (error) {
+        console.log(error);
+        console.log('transporter is not working');
+      } else {
+        console.log('All works fine, congratz!');
+      }
+    });
+
+    let mailOptions = {
+      from: 'formmaster@keepswimmingfoundation.org',
+      to: nomination.providerEmailAddress,
+      subject: 'test',
+      text: `Please verify ${nomination.verificationCode}`,
+    };
+
+    transporter.sendMail(mailOptions, function (err, data) {
```

Comment:
I would keep with fat arrow function here to be similar to other callback functions in the file: 

```javascript
transporter.sendMail(mailOptions, (err, data) => {
```